lib/capistrano/templates/puma.service.erb

```diff
@@ -0,0 +1,40 @@
+# This file tells systemd how to run Puma as a 24/7 long-running daemon.
+#
+# Customize this file based on your bundler location, app directory, etc.
+# Customize and copy this into /usr/lib/systemd/system (CentOS) or /lib/systemd/system (Ubuntu).
+# Then run:
+#   - systemctl enable <%= fetch(:puma_service_unit_name) %>
+#   - systemctl {start,stop,restart} <%= fetch(:puma_service_unit_name) %>
+#
+#
+# Use `journalctl -u <%= fetch(:puma_service_unit_name) %> -rn 100` to view the last 100 lines of log output.
+#
+[Unit]
+Description=Puma HTTP Server for <%= "#{fetch(:application)} (#{fetch(:stage)})" %>
+<%= "Requires=#{fetch(:puma_service_unit_name)}.socket" if fetch(:puma_enable_socket_service) %>
+After=syslog.target network.target
+
+[Service]
+Type=<%= service_unit_type %>
+WatchdogSec=0
+<%="User=#{puma_user(@role)}" if fetch(:puma_systemctl_user) == :system %>
+WorkingDirectory=<%= current_path %>
+ExecStart=<%= expanded_bundle_command %> exec puma -e <%= fetch(:puma_env) %>
```

Comment:
This line is different in capistrano3-puma puma 6.0.0. In the installer's `puma.service` template, there's a `-C <%= fetch(:puma_conf)` option here :thinking:. Should we add it here, or remove it from the installer?